hack/patchDevImages.sh

```diff
@@ -68,7 +68,7 @@ for val in {0..5}; do
     image_names=(webhooks keeper foghorn gc-jobs tekton-controller jenkins-controller)
 
     full_image_name=""
-    if [ -n "${registry+}" ]
+    if [ -n "${registry}" ]
```

Comment:
It should work as it's bash parameter expansion (`registry+` is a shorthand for `registry:+`) See: https://wiki.bash-hackers.org/syntax/pe
May be the error is somewhere else, let me check ...

Reply:
If `registry` is set, then `${registry+}` would be empty, `if` statement would false, and `registry` would not add to `full_image_name`.
So I still think `+` is not needed.